generator/generator.go

```diff
@@ -107,6 +106,10 @@ func generateInitRamfs(conf *generatorConfig) error {
 		return err
 	}
 
+	if conf.networkConfigType != netOff {
+		defaultModulesList = append(defaultModulesList, "kernel/drivers/net/ethernet/")
```

Comment:
`defaultModulesList` is a global variable and not suppose to be modified by a function. Activate the drivers the same way as mdraid does later:

```go
kmod.activateModules(true, true, "kernel/drivers/net/ethernet/")
```

Reply:
I've tried using `kmod.activateModules()` even before submitting this PR, but module dependencies weren't resolving correctly.
I went with new property for config, seems to work fine.